### PR TITLE
Include WPF SDK

### PR DESCRIPTION
**Pull request by dsplaisted**
Includes the WPF SDK - Microsoft.NET.Wpf.Sdk

(Note that it is not set in stone whether we'll deliver the build logic this way in the final product)

### CODE REVIEW

**Comment by dsplaisted** [comment]
@dotnet-bot test this please

File: src/redist/redist.csproj [hunk 1]
```diff
@@ -137,6 +137,12 @@
       BuildInParallel="False"
       Projects="@(SdksToBundle)">
     </MSBuild>
+
+    <!-- The WPF SDK .nupkg includes some files in the root that we don't want to bundle, so delete them -->
+    <ItemGroup>
+      <SDKFilesToDelete Include="$(ToolsetOutputDirectory)/Sdks/Microsoft.NET.Sdk.WPF/*" />
```
Comment (livarcocc) [inline src/redist/redist.csproj]: Why are the files in the nupkg to begin with? Didn't you create it?

File: build/BundledSdks.props [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <BundledSdk Include="Microsoft.NET.Sdk.Web" Version="$(MicrosoftNETSdkWebPackageVersion)" />
     <BundledSdk Include="Microsoft.NET.Sdk.Publish" Version="$(MicrosoftNETSdkPublishPackageVersion)" />
     <BundledSdk Include="Microsoft.NET.Sdk.Web.ProjectSystem" Version="$(MicrosoftNETSdkWebProjectSystemPackageVersion)" />
+    <BundledSdk Include="Microsoft.NET.Sdk.WPF" Version="$(MicrosoftNETSdkWPFPackageVersion)" />
```
Comment (livarcocc) [inline build/BundledSdks.props]: So, we are going with upper case WPF after all?

File: src/redist/redist.csproj (hunk 1 above)
Comment (dsplaisted) [reply src/redist/redist.csproj]: It's created using the corefx build system which automatically adds some files (such as a license).

File: build/BundledSdks.props (hunk 2 above)
Comment (dsplaisted) [reply build/BundledSdks.props]: No, that's a mistake (or maybe a place I forgot to update when switching to PascalCase)

**Comment by dsplaisted** [comment]
@dotnet-bot test this please

**Comment by dsplaisted** [comment]
@dotnet-bot test this please

**Comment by dsplaisted** [comment]
@dotnet-bot test this please

**Pull request merged**